src/de/animestream/src/eu/kanade/tachiyomi/animeextension/de/animestream/AnimeStream.kt

```diff
@@ -163,58 +142,30 @@ class AnimeStream : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
+        anime.description = document.select("div.desc p.f-desc").text()
+        anime.status = parseStatus(document.select("div.mvici-right span[itemprop=\"duration\"]").text())
+        anime.genre = document.select("div.mvici-left p a[rel=\"category tag\"]").joinToString(", ") { it.text() }
         return anime
     }
 
+    private fun parseStatus(status: String?) = when {
+        status == null -> SAnime.UNKNOWN
+        status.contains("Abgeschlossen", ignoreCase = true) -> SAnime.ONGOING
+        else -> SAnime.COMPLETED
+    }
+
     // Latest
 
-    override fun latestUpdatesNextPageSelector(): String = "div a.r"
+    override fun latestUpdatesNextPageSelector(): String = throw Exception("not used")
 
-    override fun latestUpdatesFromElement(element: Element): SAnime {
-        val anime = SAnime.create()
-        anime.setUrlWithoutDomain(element.select("a.tip").attr("href"))
-        anime.thumbnail_url = element.select("a.tip img").attr("src")
-        anime.title = element.select("a.tip img").attr("title")
-        return anime
-    }
+    override fun latestUpdatesFromElement(element: Element): SAnime = throw Exception("not used")
 
-    override fun latestUpdatesRequest(page: Int): Request = GET("$baseUrl/anime/?page=$page&status=&type=&order=update")
+    override fun latestUpdatesRequest(page: Int): Request = throw Exception("not used")
 
-    override fun latestUpdatesSelector(): String = "div.listupd article"
+    override fun latestUpdatesSelector(): String = throw Exception("not used")
 
     // Preferences
 
-    override fun setupPreferenceScreen(screen: PreferenceScreen) {
-        val hosterPref = ListPreference(screen.context).apply {
-            key = "preferred_hoster"
-            title = "Standard-Hoster"
-            entries = arrayOf("Streamtape", "Voe")
-            entryValues = arrayOf("https://streamtape", "https://voe.sx")
-            setDefaultValue("https://streamtape")
-            summary = "%s"
-
-            setOnPreferenceChangeListener { _, newValue ->
-                val selected = newValue as String
-                val index = findIndexOfValue(selected)
-                val entry = entryValues[index] as String
-                preferences.edit().putString(key, entry).commit()
-            }
-        }
-        val subSelection = MultiSelectListPreference(screen.context).apply {
-            key = "hoster_selection"
-            title = "Hoster auswählen"
-            entries = arrayOf("Streamtape", "Voe")
-            entryValues = arrayOf("stape", "voe")
-            setDefaultValue(setOf("stape", "voe"))
-
-            setOnPreferenceChangeListener { _, newValue ->
-                preferences.edit().putStringSet(key, newValue as Set<String>).commit()
-            }
-        }
-        screen.addPreference(hosterPref)
-        screen.addPreference(subSelection)
-    }
+    override fun setupPreferenceScreen(screen: PreferenceScreen) = throw Exception("not used")
```

Comment:
If you don't have any settings, just don't derive from `ConfigurableAnimeSource` and then there'll be no need to override this